tests/test_endpoints.py

```diff
@@ -287,6 +301,153 @@ def test_h2_grant_access_403_fail(self):
                          response.json())
 
     
+    @mock.patch("src.services.db_project_handler.upload_file_to_s3")

```

Comment:
Btw, not only endpoints should be covered with unittests, but also every separate class like DocumentsHandler and others